cypress/e2e/hub/my-imports.cy.ts

```diff
@@ -1,7 +1,21 @@
 import { randomString } from '../../../framework/utils/random-string';
-import { MyImports } from './constants';
+import { MyImports, Namespaces } from './constants';
+
+function visitImports(namespace: string) {
+  cy.navigateTo('hub', Namespaces.url);
+  cy.verifyPageTitle('Namespaces');
+  cy.getByDataCy('table-view').click();
+  cy.filterTableBySingleText(namespace);
+  // dropdown is not within row: cy.clickTableRowKebabAction(namespace, 'imports', false);
+  cy.getTableRowByText(namespace, false).within(() => {
+    cy.get('[data-cy*="actions-dropdown"]').click();
+  });
+  cy.getByDataCy('imports').click();
+
+  cy.verifyPageTitle('My imports');
+}
 
-describe('My imports', () => {
+describe.skip('My imports', () => {
```

Comment:
this will need a bugfix, skipped and opened https://issues.redhat.com/browse/AAP-29079